server/src/main/java/io/deephaven/server/table/TableModule.java

```diff
@@ -42,6 +45,11 @@
 
 @Module
 public interface TableModule {
+    @Provides
+    static TableServiceContextualAuthWiring provideAuthWiring(AuthorizationProvider authProvider) {
+        return authProvider.getTableServiceContextualAuthWiring();
+    }
```

Comment:
I notice we are a bit inconsistent IMO wrt which auth wirings we `@Provides` - looks like it's only the ones that aren't using the `AuthorizationWrappedGrpcBinding` service?

I wonder if it makes sense to have a single module that `@Provides` all the auth wirings from `AuthorizationProvider`? (And then have the impls that use AuthorizationWrappedGrpcBinding depend on the specific type instead of `AuthorizationProvider`.)

Reply:
AuthorizationWrappedGrpcBinding injects auth handlers that are invoked as gRPC delegates to the service implementation. The ContextualAuthWiring are injected more deeply into the service and do not verify authorization until Tickets have been resolved which doesn't even happen on a gRPC thread but rather the shared executor service. 

They are injected into different locations.

What would you prefer here? Is there an issue with a static `@Provides` method? In other examples like this you suggested making static methods which is why I did so here.

Reply:
We can follow-up later IMO.